src/core_modules/capture-core/rules/engine/converters/inputValueConverter.js

```diff
@@ -91,6 +91,10 @@ class RulesValueConverter implements IConvertInputRulesValue {
     convertAge(value: any): string {
         return this.convertDate(value);
     }
+
+    convertOrgUnit(value: any): string {
```

Comment:
Hey @SferaDev,

In order to avoid en `Uncaught TypeError` I suggest you either give a default value to the `value` paramether or you use a [Optional chaning ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). WHYD?
Thanks!

Reply:
Sure thing!

Reply:
Actually since none of this is typed, right now it would return ``string | undefined``. Should it throw an error if undefined? (That scenario should never happen if the input types were enforced) Or it should return ``string | undefined`` even though the rest of the methods do not handle this ``undefined`` scenario?

Reply:
I would propose to follow the pattern of the rest of the methods in the class. If you look at `convertUrl` or `convertEmail`, they both return `''` for `falsey` values (`undefined` included).  

In your case it can become:
```
    convertOrgUnit(value: any): string {
        return value?.id || '';
    }
```

WDYT?

Reply:
Sure thing, I see you're not using the Nullish coalescing operator (??) so I'll update now with your proposal. Thanks @simonadomnisoru!